src/functions/blanket/blanket_heat.jl

```diff
@@ -0,0 +1,32 @@
+c1 = 14.26;
+c2 = -0.05675;
+
+a =   4.132e+04;
+b =    -0.08001;
+c =   -4.13e+04;
+d =    -0.08002;
+
+TargetHeatLeavingBlanket = .00004;
+
+Pw = 4;
+
+# fitFunc = @(x) c1 * exp(c2 * x);
+f(x) = a*exp(b*x) + c*exp(d*x);
+
+xx =  linspace(0,200,1001);
+localHeatingNorm = f(xx);
```

Comment:
// wrap in own func (local_heating that is). then all these constants don't obfuscate what's going on